tool-metagen/src/DA/XML/Schema/Convert.hs

```diff
@@ -664,6 +664,7 @@ mkFieldType QName{..} =
         "any"                | isXsd -> Prim PrimText
 
         "decimal"            | isXsd -> Prim PrimDecimal
+        "double"             | isXsd -> Prim PrimDecimal
```

Comment:
Going from floating point to fixed-point is probably a bad idea ... Are there any checks to ensure the conversion is sane?

Reply:
The checks would occur at ingestion time (or better yet via a specific library designed for it such as `lib-integration-java`), but I think that for now (given competing priorities) storing it as a Decimal is the right approach. If we're being pedantic, and XML Decimal is not a Daml Decimal.